Lib/multiprocessing/popen_spawn_win32.py

```diff
@@ -32,8 +38,12 @@ class Popen(object):
     def __init__(self, process_obj):
         prep_data = spawn.get_preparation_data(process_obj._name)
 
-        # read end of pipe will be "stolen" by the child process
+        # read end of pipe will be duplicated by the child process
         # -- see spawn_main() in spawn.py.
+        #
+        # bpo-33929: Previously, it was "stolen" by the child process, but it
+        # leaked a handle if the child process has been terminated before it
+        # "stole" the handle (and closed it in the parent process).
```

Comment:
"before it could steal the handle from the parent process"

Reply:
fixed